release.yaml

```diff
@@ -151,4 +151,3 @@ api:
     obr: true
     bom: true
     mvp: true
-    isolated: true
```

Comment:
This happened last time for some reason the script cuts off this line which we need at the end - can you add back in please? And look at fixing that error in the script for next time pls (can be tomorrow)?